### ISSUE TITLE
Mandatory field star not populating on @combobox 

### ISSUE THREAD

**Issue by addulaajayreddy** [openanthem/nimbus-core#521]
# Issue Details

**Type of Issue**  (check one with "X")
```
[ X ] Bug Report => Please search GitHub for a similar issue or PR before submitting
[ ] Feature Request => Please ensure feature is not already in progress
[ ] Support Request => Please do not submit support requests here, instead see: https://discourse.oss.antheminc.com/
```

## Current Behavior
Combobox mandatory icon pops up on event 

## Expected Behavior
Mandatory star on combobox should be displayed on the launch of form.

### Steps to Reproduce
create and combobox inside a form and make it mandatory.
Launch the form on browser. the field is populated but not the start icon.
On selecting the dropdown the start icon appears.  

### Code Snippet
@MapsTo.Path
@Model.Param.Values(value= SelectBox.class)
@NotNull
@ComboBox(postEventOnChange =true)
@Label("Test Check Box")
private String purpose;

# Environment Details
* **Nimbus Version:**
1.3.1.M1

* **Browser:**
Google Chrome

**Comment by lopezton-anthem** [comment]
@addulaajayreddy a fix has been deployed in Nexus Staging to 1.3.1.M1.  Can you please verify if it is working now?

**Comment by addulaajayreddy** [comment]
@lopezton-anthem Working as expected. Thanks for the quick turn around
